Analyzer/plugins/Analyzer.cc

```diff
@@ -2733,7 +2732,7 @@ bool Analyzer::passPreselection(const reco::TrackRef track,
         pf_isNeutHadron = pfCand->translatePdgIdToType(pfCand->pdgId()) == reco::PFCandidate::ParticleType::h0;
         pf_isUndefined = pfCand->translatePdgIdToType(pfCand->pdgId()) == reco::PFCandidate::ParticleType::X;
         pf_isPfTrack = true;
-        pf_energy = pfCand->ecalEnergy() + pfCand->hcalEnergy();
+        //pf_energy = pfCand->ecalEnergy() + pfCand->hcalEnergy();
```

Comment:
I use this to calc EoP why are you commenting this out? :D

Reply:
Because I use energies from isolation map and not the PF candidate energy

Reply:
well two things:
1) I think the group was going towards not using the Iso but the PF
2) if you use the iso map energy, why did you comment out a variable that's called *pf* ?